src/pages/ResourceForm.tsx

```diff
@@ -90,9 +96,34 @@ const ResourceForm: FC<ResourceFormProps> = ({ uppy, resource, resourceType }) =
       features: Yup.object().shape({
         dlr_title: Yup.string().required(t('feedback.required_field')),
       }),
+      licences: Yup.array().of(
+        Yup.object().shape({
+          identifier: Yup.string().required(t('feedback.required_field')).min(1, t('ikke tom id')),
+          features: Yup.object().shape({
+            dlr_non_existent_field: Yup.string().required(t('Bør feile!')),
+          }),
+        })
+      ),
```

Comment:
Har knotet en del med validering av arrays selv ja, men du må i hvert fall angi hvor mange elementer som er obligatorisk:
```suggestion
      ).min(1, "DU MÅ HA EI FIL!!!"),
```
(du må kanskje trikse litt ekstra med `touched`, siden det bare settes for felter inni arrayet, og ikke arrayet selv, men tror det skal gå greit siden touched vil bli satt til `[...]` som uansett er true...)

Reply:
touched funker  (men vi setter manuelt) - men jeg tror vi sliter med at vi bruker et objekt i array og ikke bare en string

Reply:
hm, det ligner jo veldig på hvordan vi validerer filer: 1 fil er obligatorisk og alle filer må ha en valgt lisens: 
https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/src/utils/validation/registration/registrationValidation.ts#L49

ser nå at vi da måtte manuelt sjekke at det ikke var lagt til noen filer enda: https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/src/pages/registration/FilesAndLicensePanel.tsx#L107

Reply:
Det høres us som om jeg må ha manuell validering da. Det kan jeg iallefall få til :-)

Reply:
Problemet var typo i yup-oppsettet. Men jeg endte opp med å ha både manuell onBlur og manuell visning av error.